ExampleApp/src/components/AssetViewerBox.tsx

```diff
@@ -0,0 +1,160 @@
+import { H5, IOColors, hexToRgba } from "@pagopa/io-app-design-system";
```

Comment:
what about adding a top level comment on each file to briefly explain its purpose?